c_tests/tests/debug_debuginfo.c

```diff
@@ -1,19 +1,4 @@
-// Compiler:
-// Run-time:
-//   env-var: YKD_PRINT_IR=jit-pre-opt
-//   stderr:
-//     ...
-//     define internal void @__yk_compiled_trace_0(i32* %0) {
-//       %2 = alloca i8*, align 8
-//       ; main() tests/debug_debuginfo.c:28:9
-//       store i8* null, i8** %2, align 8, !dbg !3
-//       ; main() tests/debug_debuginfo.c:29:7
-//       store i32 2, i32* %0, align 4, !dbg !12
-//       ; main() tests/debug_debuginfo.c:30:37
-//       %3 = load i8*, i8** %2, align 8, !dbg !13
-//       ret void
-//     }
-//     ...
+// ignore: impractical to test all optimisation levels.
```

Comment:
Couldn't we have separate test files for the different opt levels?

Reply:
We could, but I didn't see the benefit in checking exact position of the comments and the line and column numbers in the comments for every single opt level. Especially since debuginfo is approximate.

Reply:
I could make one more generic test which just checks if there is at least on debug comment in the IR. How about that?

Reply:
Let's say the current test is the O0 test, why not put it in `debug_debuginfo.c.O0` as-is?

Reply:
I don't know what you mean.

Do you mean paste the verbatim IR for each optimisation level into a different file?

What would be the value of such a specific test?

Reply:
Note that this PR already introduces a`debug_debuginfo.c.O0` which does test the verbatim IR.

I just don't see the worth in testing the verbatim IR of approximate debuginfo for all opt levels. It will be a pain to update when things change.

Reply:
Mea culpa! You've already done what I wanted but I was too stupid to realise.

Reply:
Ah, OK!